### DISCUSSION TITLE
True/False Form Field

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#6257] [Help]
Using MySQL as a datasource for a true/false field, the field is of type TINYINT(1), rather than a Boolean. When I create screens from that table, the form field is automatically a 'Number Field'. When I attempt to add an option picker I cannot set this field as being tied to that column. It appears there is not 'toggle - ON/OFF' button element commonly referred to as a Toggle Switch. Default value would be 0 for false under any circumstance. Is there any plan to introduce such a form element? Ideally it should be set up similar to an option picker with a field label, then a label-value for false and true.

![image](https://user-images.githubusercontent.com/106758851/172942597-20a9a31d-1322-47db-b5de-74ad66739678.png)

**Comment by melohagan** [comment]
Hi @mmalessa80 

You could create a button group for that field, similar to this: https://github.com/Budibase/budibase/discussions/5237#discussioncomment-2501601
